app/Http/Controllers/AlbumController.php

```diff
@@ -186,6 +187,21 @@ public function setDescription(SetAlbumDescriptionRequest $request): void
 		$request->album()->save();
 	}
 
+	/**
+	 * Change the copyright of the album.
+	 *
+	 * @param SetAlbumCopyrightRequest $request
+	 *
+	 * @return void
+	 *
+	 * @throws ModelDBException
+	 */
+	public function setCopyright(SetAlbumCopyrightRequest $request): void
+	{
+		$request->album()->copyright = $request->copyright();
```

Comment:
You need to add a `@property` annotation to the BaseAlbumImpl and BaseAlbum description.

Reply:
Will look into the above issues